app/pages/admin/project-status.jsx

```diff
@@ -85,10 +86,11 @@ class ProjectStatus extends Component {
     this.setState({ dialogIsOpen: false });
   }
 
-  handleDialogSuccess(workflow) {
+  handleDialogSuccess() {
+    const defaultWorkflow = this.state.workflows.filter(workflow => workflow.id === this.state.defaultWorkflowId);
     this.state.project.update({ 'configuration.default_workflow': undefined }).save()
       .then(() => {
-        workflow.update({ active: false }).save()
+        defaultWorkflow[0].update({ active: false }).save()
```

Comment:
I changed the name of this variable because of a linter error (`'workflow' is already declared in the upper scope`)